CHANGELOG.md

```diff
@@ -1,6 +1,26 @@
 # Unreleased
-- Added additional attributes to ochttp spans
 
+- None
+
+# 3.8.7
+
+## Added
+
+- Url parameters can be used in the target definition
```

Comment:
pls add `Thanks to @XXX` for external contributirs

Reply:
Of course!